src/main/java/core/basesyntax/ColorSupplier.java

```
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random rand = new Random();
+        int choice = rand.nextInt(3);
+        switch (choice) {
+            case 0:
+                return Colors.RED.toString();
+            case 1:
+                return Colors.GREEN.toString();
+            case 2:
+                return Colors.BLUE.toString();
+            default:
+                return "";
```

Returning `""` in the default branch hides type issues and is unnecessary if you index into `Colors.values()` correctly. Instead, return a `Colors` value (after changing the method signature) or throw an exception if something truly unexpected happens. The default branch should not return an empty string for a color supplier.